include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -231,7 +342,26 @@ class NBL_API IDescriptorSetLayout : public virtual core::IReferenceCounted
 			return true;
 		}
 
+		inline size_t getTotalMutableSamplerCount() const { return m_mutableSamplerCount; }
+		inline size_t getTotalDescriptorCount(const E_DESCRIPTOR_TYPE type) const { return m_descriptorCount[type]; }
```

Comment:
if all your offsets are `uint32_t` then these should be `uint32_t` too

Reply:
Resolved.